### DISCUSSION TITLE
Weekly Report (2023-11-05)

### DISCUSSION THREAD

**Discussion by drafish** [PlanckerLabs/remix-cc#102] [Meeting of Remix CC]
### 通知
本着少开会多做事的原则，Remix CC 取消每周固定周会，平时通过异步沟通来同步进度，如若需要可随时安排会议

**Comment by drafish** [comment]
### DJQ
1. 与 yiwen 交接主理人相关工作
2. 与星火链网开了 Remix 插件项目启动会
3. 与 Remix 同步近期的进度

**Comment by yiwen4** [comment]
### yiwen
1.交接主理人相关工作
2.梳理RCC规划、Q3总结、10月工作量汇总、提交Q4申请
